server/routes/walletRouter.spec.js

```diff
@@ -64,6 +64,27 @@ describe("walletRouter", ()=> {
       expect(res.body.wallets[0]).property("tokens_in_wallet").eq(10);
     });
 
+    it("should omit private fields", async () => {
+      const wallet = new Wallet({
+        "id":  uuid.v4(),
+        "type": "p",
+        "name": "name",
+        "password": "private field",
+        "salt": "private field",
+        "tokens_in_wallet": 10
+    })
+      sinon.stub(WalletService.prototype, "getById").resolves(wallet);
+      sinon.stub(TrustService.prototype, "convertToResponse").resolves(mockTrust);
+      sinon.stub(TokenService.prototype, "countTokenByWallet").resolves(10);
+      sinon.stub(Wallet.prototype, "getSubWallets").resolves( []);
+      const res = await request(app)
+        .get('/?limit=2');
+      expect(res).property("statusCode").eq(200);
+      expect(res.body.wallets).lengthOf(1);
+      const resWallet = res.body.wallets[0]
+      expect(resWallet).not.to.contain.keys(['password', 'salt', 'type'])
+    });
+
```

Comment:
This is an interesting case, I thought.
Do you think we can just test the omit function in util (against the fn directly), for covering the code? In that way, it would be more easy?
I am considering using util/model pieces, and just write unit test for them, to bring some relief for writing tests.

Reply:
@deepwinter is this your opinion you ever mentioned?

Reply:
@dadiorchen I think we could use a ready made omit function like with a lodash library and then this test would stay the same. In my opinion, it's more important to have a test that checks that the fields are omitted (aka behaviour test) than test implementation - omit function. Tell me what you think.
PS I've noticed that there are minimal amount of dependencies in the project, how do you feel about importing new libraries? like lodash?

Reply:
I definitely support he use of lodash in all our node.js services.

Reply:
I didn't use lodash, but I'd like to try it, so feel free to add it.
About writing the test, these days I'm thinking of the strategy, my thoughts on this is that we use unit tests to cover edge cases, and use integration test to cover major workflow, because the latter is more heavy bringing burden for devs.

Reply:
Oh, this isn't integration, but my point is that use the most easy way to cover code.

Reply:
cool, I've added lodash and now it's nicer. It's super easy to use and very handy

Reply:
I can't merge this though (no write access), so unless any changes required, please feel free to merge @dadiorchen or @deepwinter